app/app/User.php

```diff
@@ -2,32 +2,28 @@
 
 namespace App;
 
-use Eloquent;
-use Illuminate\Support\Facades\Hash;
-use DB;
```

Comment:
You removing the "use DB" but the hasRole method still references DB.

Fix this.